apps/openmw/mwmechanics/aistate.hpp

```diff
@@ -38,6 +38,16 @@ namespace MWMechanics
             //return a reference to the (new allocated) object 
             return *result;
         }
+
+        template< class Derived >
+        Derived* copy() const
```

Comment:
Why don't use `std::unique_ptr`?

Reply:
The rest of file does not use it.

Reply:
I don't see a point. Safe code is much more important than type usage in some file.